app/javascript/components/ListingAddress.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import { RailsListing } from "../modules/listings/SharedHelpers"
+
+export interface ListingAddressProps {
+  listing: RailsListing
+}
+
+export const ListingAddress = ({ listing }: ListingAddressProps) => {
+  return (
+    listing.Building_Street_Address &&
+    listing.Building_City &&
+    listing.Building_State &&
+    listing.Building_Zip_Code && (
+      <span>
+        {listing.Building_Street_Address}, {listing.Building_City},{" "}
+        <abbr title="California">CA</abbr> {listing.Building_Zip_Code}
```

Comment:
if we wanna nix the annoying text decoration here, I think you can add `className="no-underline"` to the `abbr`. and sorry for not adding on my PR 😭